src/helper/main.js

```diff
@@ -79,6 +79,74 @@ import { logAndDie, helpMessage, databases } from './helperUtil';
             console.log('Finished!');
             break;
         }
+        case 'firebase': {
+            try {
+                console.log('Reading info...');
+                const info = JSON.parse(await fs.readFile('./data/firebase.json'));
+                const ownerOrKeyRegex = /^(?:\w+-\w+-\w+)|(?:owner)$/;
+                const {
+                    discordUserData: users,
+                    discord: { autoresponse, cooldowns },
+                } = info;
+                console.log('Converting users...');
+                const newUsers = Object.entries(users).map(([id, { xp, faction, ...data }]) => {
+                    let key = null;
+                    let giveaways = '';
+                    if (data.key) {
+                        if (data.key.match(ownerOrKeyRegex)) key = data.key;
+                        else giveaways = data.key;
+                    }
+                    const user = new UserDto(
+                        xp,
+                        undefined,
+                        faction,
+                        0,
+                        0,
+                        key,
+                        undefined,
+                        undefined,
+                        giveaways
+                    );
```

Comment:
use the default values instead.
iirc one is able to say giveaways=giveaways in js too etc

Reply:
Thats not how js works